source/nanoFramework.Json.Test/nanoFramework.Json.Test.nfproj

```diff
@@ -41,9 +41,6 @@
       <SpecificVersion>True</SpecificVersion>
     </Reference>
   </ItemGroup>
-  <ItemGroup>
-    <None Include="packages.config" />
-  </ItemGroup>
```

Comment:
Why has this been removed?

Reply:
Not sure why.  I didn't intentionally remove it.